src/app/public/modules/datepicker/datepicker-calendar.component.ts

```diff
@@ -66,7 +66,10 @@ export class SkyDatepickerCalendarComponent implements AfterViewInit {
     return this.adapter.elementIsVisible();
   }
 
-  @ViewChild(SkyDatepickerCalendarInnerComponent)
+  @ViewChild(SkyDatepickerCalendarInnerComponent, {
+    read: SkyDatepickerCalendarInnerComponent,
+    static: true
```

Comment:
☝️ The `static: true` is to force the overlay service to render the `SkyDatepickerCalendarInnerComponent` component during `ngOnInit` so it's present before logic is ran on it in the `writeValue` method below.